src/views/studio/StudioLayout.tsx

```diff
@@ -47,25 +45,24 @@ const StudioLayout = () => {
   const location = useLocation()
   const displayedLocation = useVideoEditSheetRouting()
   const { isUserConnectedToInternet, nodeConnectionStatus } = useConnectionStatus()
-  const { extensionConnected: extensionStatus, accounts } = useJoystream()
 
   const {
-    activeUser: { accountId, memberId, channelId },
-    loading: activeUserLoading,
-  } = useActiveUser()
-
-  const { memberships, loading: membershipsLoading } = useMemberships({
-    where: { controllerAccount_in: accounts.map((a) => a.id) },
-  })
+    activeAccountId,
+    activeMemberId,
+    activeChannelId,
+    extensionConnected: extensionStatus,
```

Comment:
```suggestion
    extensionConnected
```
Analogous to StudioEntrypoint. Remove line 58 and then on the line 63: 
`const accountSet = !!activeAccountId && !!extensionConnected`
I think it's better